anet-dictionary.yml

```diff
@@ -27,7 +27,15 @@ reportWorkflow:
   nbOfHoursQuarantineApproved: 24
   nbOfHoursApprovalTimeout: 48
 
-maxTextFieldLength: 250
+# Add exceptions if there is any, only `default` is mandatory, others are optional
+maxTextFieldLength:
+  default: 250
+  report:
+    nextSteps: 1000
```

Comment:
I suggest defining this value in the definition of the field instead of a central place for text field lenths

Reply:
And for the default value, we use the older version?

Reply:
This also requires us to add additional fields to dictionary, for `authorizationGroups` for example, there are no fields, we will need to enable those in the schema as well. That is a lot of change for a simple issue